Library/Homebrew/cmd/uninstall.rb

```diff
@@ -1,3 +1,9 @@
+#:  * `rm`, `remove`, `uninstall` [`--force`] <formula>:
```

Comment:
I think it would be a good idea to make the canonical name (`uninstall`) the first and list the aliases (`rm`, `remove`) afterwards. Otherwise the order in the generated man page will be weird sometimes. (This is the command where this discrepancy is most obvious.)